dev-lua/tolua++/tolua++-1.0.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="An extension of toLua, a tool to integrate C/Cpp code with Lua"
+DESCRIPTION="tolua++ is an extension of toLua, a tool to integrate C/C++ code with \
+Lua. tolua++ includes new features oriented to c++, such as class templates."
+HOMEPAGE="https://github.com/LuaDist/toluapp"
+COPYRIGHT="2009 Ariel Manzur"
+LICENSE="MIT"
+REVISION="1"
+COMMIT_SHA="b34075b76835b778bb6b2ce0aa224afd9d182887"
+SOURCE_URI="https://github.com/LuaDist/toluapp/archive/$COMMIT_SHA.zip"
+CHECKSUM_SHA256="c69a99a4b0a2bb3decab09bd8d2473911419bef601c86bb6c00ff670f88b4bb1"
+SOURCE_DIR="toluapp-$COMMIT_SHA"
+
+ARCHITECTURES="x86_64"
```

Comment:
Only x86_64? Could you check if it's possible to build for x86_gcc2 or x86 (secondary architecture)?

Reply:
@timfel, @Begasus: good news: it builds fine on x86_gcc2!
```
getarch && tolua++ -v
x86_gcc2
tolua++-1.0.92 (written by W. Celes, A. Manzur)
```
And I'm confident that it will also work fine on x86.
BTW, since this recipe provides libs, it would be nice to enable support for x86 secondary arch.
(I'll write another review with the changes it.)